src/LiveDevelopment/Agents/RemoteFunctions.js

```diff
@@ -319,6 +336,14 @@ function RemoteFunctions(experimental) {
             if (this.trigger) {
                 _trigger(element, "highlight", 1);
             }
+            
+            if (!window.event && !isInViewport(element)) {
```

Comment:
I am checking if there is any window event (mostly click event) is there on the LivePreview instance, then we don't need to bring the element in viewport, as it is already present in the viewport.